core/src/commands/preview.tsx

```diff
@@ -91,3 +91,32 @@ export const codeLive: ICommand = {
     }
   },
 };
+
+export const editablePreview: ICommand = {
```

Comment:
https://github.com/uiwjs/react-md-editor/blob/2520e88123bde4f4d31415bb362183964f94ca48/core/src/index.tsx#L6-L9

```jsx
export { editablePreview } from './commands/preview';
```

@vishwamartur  It is necessary to export editablePreview in the entry point.